recipes/server.rb

```diff
@@ -0,0 +1,87 @@
+# Centos7 comes with mariadb-libs, which conflicts with mysql
```

Comment:
it looks a lot like what mysql cookbook offers. Could you check if you couldn't use that?